templating/twig_extension_repository.rst

```diff
@@ -0,0 +1,93 @@
+.. index::
+   single: Using the Twig Extensions Repository
+
+How to Use the Twig Extensions Repository
+=========================================
+
+The `Twig official extensions repository`_ contains some
```

Comment:
Twig official extensions repository (or change the target at the end of the document)